src/main/java/com/google/devtools/build/lib/graph/DFS.java

```diff
@@ -98,7 +95,8 @@ public void visit(Node<T> node, GraphVisitor<T> visitor) {
     Collection<Node<T>> edgeTargets = transpose
         ? node.getPredecessors() : node.getSuccessors();
     if (edgeOrder != null) {
-      List<Node<T>> mutableNodeList = Ordering.from(edgeOrder).sortedCopy(edgeTargets);
+      List<Node<T>> mutableNodeList =
```

Comment:
@laszlocsomor The reason I decided to replace `Ordering` here with streams is because it seems that `Ordering` is now [obsolete for Java 8 users](https://google.github.io/guava/releases/22.0/api/docs/com/google/common/collect/Ordering.html) (for the most part).

If you think this change isn't worth it, please let me know and I'd be happy to revert it. :)